fares-data-build-tool/fdbt-site
# feat: added hint text to search operators page

# Description

-   This PR adds hint text to the search operators page to explain to a user what a noc code is.

# Testing instructions

-   Pull down this branch and run the site locally. Check that the extra help text is present on the search operators page and that it reads correctly.

# Type of change


# Checklist:
